### PR TITLE
Remove duplicate type definitions from function signatures

### PR DESCRIPTION
**Pull request by paskal**
I haven't found a linter for these, so I had to catch these manually. I found #757 to fix one of these, and I thought it would be good to fix everything at once.

### CODE REVIEW

File: backend/app/rest/api/ssl.go [hunk 1]
```diff
@@ -111,15 +111,11 @@ func (s *Rest) getRemarkHost() string {
 
 func (s *Rest) makeTLSConfig() *tls.Config {
 	return &tls.Config{
-		PreferServerCipherSuites: true,
```
Comment (paskal) [inline backend/app/rest/api/ssl.go:114]: This is not used since 1.17, https://github.com/golang/go/commit/9d0819b27ca248f9949e7cf6bf7cb9fe7cf574e8

**Comment by VladimirZaets** [comment]
@paskal I personally prefer full declaration:)

**Comment by umputun** [comment]
this pr is a little bit confusing. I don't know how safe the part about TLS change (will need to check) but  for sure it is not a part of "remove duplicate definition from func"

can you pls keep only those lint related changes here and make a separate pr for tls config?

**Comment by paskal** [comment]
@umputun done, feel free to re-review.

**Review by umputun — APPROVED** [review]

**Pull request merged**